docs/release-notes.rst

```diff
@@ -72,6 +72,12 @@ For each release, we list the following types of change (in this order):
    in the "stable" docs (corresponding to the last release) but will be visible in the
    "latest" docs (corresponding to the master branch).
 
+Bug Fixes
+---------
+
+- Parallel execution introduced in 0.8.0 had a bug in logging, where logging breaks
```

Comment:
"introduced in 0.8.0" should probably be in parentheses (or bracketed by commas). The way it is now, it looks restrictive, as if there were multiple kinds of parallel execution.